mmdt/blog/migrations/0005_alter_comment_active.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 4.2.4 on 2024-02-08 05:19
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('blog', '0004_post_view_count'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='comment',
+            name='active',
+            field=models.BooleanField(default=True),
+        ),
+    ]
```

Comment:
Just confirmation , 
Is this migration only for setting up the "comment active" column to be set as Default True? 
Could we simply set active=True when sending a comment in a blog post without going through this migration? Would this approach require redoing the migration when we want to turn off the "comment active" to False?

Reply:
I just fixed it, can you please check here>>>
- https://github.com/khinezarthwe/mmdt_web_app/pull/77/commits/d6a9d0d44bf171025dd17f0e8d36e80d7d7c87bd

Reply:
Yes, It work well., then we don't need that migration file, Please remove this migration file mg lay.